WordPress/wporg-main-2022
# Update edge spacing to match header/footer

See https://github.com/WordPress/wporg-parent-2021/issues/65. The spacing to the left and right of content in the header & footer was updated to be 80px on large screens, and snap down to 20px on small-medium screens (<890). This left the page content out of alignment. In this PR, I'm fixing that for the main theme.

### Screenshots

Screenshots across 3 screen sizes.

Note: 1280 is the full page, the others cut off at 1200px tall.

<table role="table">
<thead>
<tr>
<th></th>
<th>1280</th>
<th>960</th>
<th>480</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td>Home</td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969798-34d75208-b337-4c6a-87f2-8ffeedaa7802.png"><img src="https://user-images.githubusercontent.com/541093/207969798-34d75208-b337-4c6a-87f2-8ffeedaa7802.png" alt="branch-home-1280" style="max-width: 100%;"></a></td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969797-87647dcd-3dac-401e-b4f5-0d15ac8c87a8.png"><img src="https://user-images.githubusercontent.com/541093/207969797-87647dcd-3dac-401e-b4f5-0d15ac8c87a8.png" alt="branch-home-960" style="max-width: 100%;"></a></td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969793-f89fb2d6-e067-4f38-8462-d2f89d6bb4b4.png"><img src="https://user-images.githubusercontent.com/541093/207969793-f89fb2d6-e067-4f38-8462-d2f89d6bb4b4.png" alt="branch-home-480" style="max-width: 100%;"></a></td>
</tr>
<tr>
<td>Mobile</td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969807-a6ee1791-2e72-4444-a160-6373cab0ad35.png"><img src="https://user-images.githubusercontent.com/541093/207969807-a6ee1791-2e72-4444-a160-6373cab0ad35.png" alt="branch-mobile-1280" style="max-width: 100%;"></a></td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969806-7fab1a19-a5cf-43db-b0ab-75f0749ea0c1.png"><img src="https://user-images.githubusercontent.com/541093/207969806-7fab1a19-a5cf-43db-b0ab-75f0749ea0c1.png" alt="branch-mobile-960" style="max-width: 100%;"></a></td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969800-217a62f6-9e98-419b-8faf-0f572dd6ea1f.png"><img src="https://user-images.githubusercontent.com/541093/207969800-217a62f6-9e98-419b-8faf-0f572dd6ea1f.png" alt="branch-mobile-480" style="max-width: 100%;"></a></td>
</tr>
<tr>
<td>Download</td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969791-d9432541-c91e-4240-9d47-133d6b5b7957.png"><img src="https://user-images.githubusercontent.com/541093/207969791-d9432541-c91e-4240-9d47-133d6b5b7957.png" alt="branch-download-1280" style="max-width: 100%;"></a></td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969788-fcd881fe-bffb-4182-bb0a-15e3d1183fc4.png"><img src="https://user-images.githubusercontent.com/541093/207969788-fcd881fe-bffb-4182-bb0a-15e3d1183fc4.png" alt="branch-download-960" style="max-width: 100%;"></a></td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969784-a47dab92-73d6-4b57-b68d-246943712843.png"><img src="https://user-images.githubusercontent.com/541093/207969784-a47dab92-73d6-4b57-b68d-246943712843.png" alt="branch-download-480" style="max-width: 100%;"></a></td>
</tr>
<tr>
<td>Download - Counter</td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969774-f41ceba5-d940-4b67-8e84-5c558d3a53e2.png"><img src="https://user-images.githubusercontent.com/541093/207969774-f41ceba5-d940-4b67-8e84-5c558d3a53e2.png" alt="branch-dl-counter-1280" style="max-width: 100%;"></a></td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969773-e5562e0b-e4e6-4b11-a25f-6c7ccee5187e.png"><img src="https://user-images.githubusercontent.com/541093/207969773-e5562e0b-e4e6-4b11-a25f-6c7ccee5187e.png" alt="branch-dl-counter-960" style="max-width: 100%;"></a></td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969768-e4eee1ee-c51f-4156-82ec-1e3bc67898df.png"><img src="https://user-images.githubusercontent.com/541093/207969768-e4eee1ee-c51f-4156-82ec-1e3bc67898df.png" alt="branch-dl-counter-480" style="max-width: 100%;"></a></td>
</tr>
<tr>
<td>Download - Releases</td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969782-7858577f-34a1-4d0d-8064-22124e507ab6.png"><img src="https://user-images.githubusercontent.com/541093/207969782-7858577f-34a1-4d0d-8064-22124e507ab6.png" alt="branch-dl-releases-1280" style="max-width: 100%;"></a></td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969780-c89b43b2-8683-4e94-aed8-34aed6f4d695.png"><img src="https://user-images.githubusercontent.com/541093/207969780-c89b43b2-8683-4e94-aed8-34aed6f4d695.png" alt="branch-dl-releases-960" style="max-width: 100%;"></a></td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/541093/207969775-b2b1e5ac-a79f-4544-9b8d-01ef4733689c.png"><img src="https://user-images.githubusercontent.com/541093/207969775-b2b1e5ac-a79f-4544-9b8d-01ef4733689c.png" alt="branch-dl-releases-480" style="max-width: 100%;"></a></td>
</tr>
</tbody>
</table>

### How to test the changes in this Pull Request:

1. Build the branch `yarn workspaces run build`
2. View the pages
3. The content should line up with the "News" header item and the first footer column
4. When you scale the page below 890px, the header switches to the small header and the content should align with the WordPress logo
5. Spot check that no other alignments are broken
